roles/custom/matrix-bot-draupnir/templates/production.yaml.j2

```diff
@@ -261,4 +258,4 @@ pollReports: false
 
 # Whether or not new reports, received either by webapi or polling,
 # should be printed to our managementRoom.
-displayReports: false
+displayReports: {{ matrix_bot_draupnir_abuse_reporting_enabled }}
```

Comment:
It's probably better to add a new variable for this. Its default value can be defined in terms of `matrix_bot_draupnir_abuse_reporting_enabled`.

Based the setting key, the variable should be named `matrix_bot_draupnir_display_reports`.

Perhaps `abuseReporting.enabled` should also receive its own variable (or use `matrix_bot_draupnir_abuse_reporting_enabled`).

Then, `web.enabled` can be its own variable too, with a default defined in terms of `matrix_bot_draupnir_abuse_reporting_enabled`:

```yml
matrix_bot_draupnir_web_enabled: "{{ matrix_bot_draupnir_abuse_reporting_enabled }}"
```

`matrix_bot_draupnir_abuse_reporting_enabled` should probably be renamed to `matrix_bot_draupnir_web_abuse_reporting_enabled`.

All these variables names are meant to indicate where the variable is being used. We usually try to make variables take their names from the setting path that they control.

-----

All this would let people:

- enable the web-interface manually (for other purposes), without enabling abuse-reporting
- control `displayReports` separately (being able to turn it off, even if abuse reporting is enabled)